modules/statics/src/coins/erc20Coins.ts

```diff
@@ -786,7 +807,11 @@ export const erc20Coins = [
     'Blockchain.io',
     18,
     '0xcdc412f306e0c51e3249b88c65423cd16b322673',
-    UnderlyingAsset.BCIO
+    UnderlyingAsset.BCIO,
+    AccountCoin.getFeaturesByTypeExcluding(
+      [CoinFeature.CUSTODY_BITGO_GERMANY, CoinFeature.CUSTODY_BITGO_EUROPE_APS, CoinFeature.CUSTODY_BITGO_FRANKFURT],
+      ETH_FEATURES
```

Comment:
I am able to see that some of these tokens didn't have ETH_FEATURES before but now we are adding ETH_FEATURES, should be fine though  mostly as they are eth tokens